spec/support/fake_rails.rb

```diff
@@ -1,9 +1,10 @@
 require "rack"
+require "ostruct"
```

Comment:
(I think ActiveSupport was requiring it before, so everything worked. ActiveSupport 4 appears not to, so we get a NameError if we don't add our explicit require.)